### PR TITLE
Add Networking Basics Article

### PR DESCRIPTION
**Pull request by dlford**
Love what you're doing here! I've added my Introduction to Networking article, I think it'll be a nice addition to this resource.

### CODE REVIEW

File: docs/networking/networking-basics.md [hunk 1]
```diff
@@ -0,0 +1,520 @@
+# Networking Basics
+
+> Written by Dan Ford and originally published on <a href="https://dlford.io/computer-networking-binary-hex-explained/" target="_blank" rel="noopener noreferrer">dlford.io</a>.<br />
+> Copyright &copy; 2019 dlford.io all rights reserved.
```
Comment (BCNelson) [inline docs/networking/networking-basics.md]: Sorry I don't think we can take this article into the wiki with the attached copyright

File: docs/networking/networking-basics.md [hunk 2]
```diff
@@ -0,0 +1,520 @@
...
+  - Type the command `ipconfig | findstr /i "Gateway"`
+- Mac
+  - Go to Applications > Utilities, and double click on Terminal
+  - Type the command `netstat -nr | grep default`
+- Linux
+  - Open a terminal shell, the method for this varies by distribution, on Ubuntu you can use the hotkey `control` + `shift` + `t`
+  - Type the command `ip route | grep default`
+
+You should get a value of something similar to `192.168.0.1`, this is the IP address of your gateway. Open a web browser and put this number in the address bar with `http://` in front of it, for example `http://192.168.0.1`
+
```
Comment (py-crash) [inline docs/networking/networking-basics.md]: Some router will only admit calls in https and will return an error if called with http. It's an edge case, I now, however I should be incorporated here.

File: docs/networking/networking-basics.md [hunk 3]
```diff
@@ -0,0 +1,520 @@
...
+You will need to log in to this device, if you don't have the username and password, it may be on a sticker on the bottom of the unit (you should also change this after logging in to make your network more secure!). If it's not on a sticker, you'll have to look in the user manual for your modem or router, or search the web for "_(model number of your modem or router) default password_".
+
```
Comment (py-crash) [inline docs/networking/networking-basics.md]: There should be a big warning here that tells you that if you are using the default password you should change it

File: docs/networking/networking-basics.md [hunk 4]
```diff
@@ -0,0 +1,520 @@
+# Networking Basics
```
Comment (RealOrangeOne) [inline docs/networking/networking-basics.md]: Meta question: How useful is it to have a page on a self hosting wiki which is, arguably, going into some pretty low level networking fundemental.

Besides the different between public/private V4 address space, very little has come up for me when self-hosting.

File: docs/networking/networking-basics.md (hunk 1 above)
Comment (RealOrangeOne) [reply docs/networking/networking-basics.md]: Unfortunately, i'm inclined to agree. 

That being said, if you wanted to thin this post down (a la https://github.com/selfhostedshow/wiki/pull/7/files#r382993359), and link through to your article for further details, I think i'd be ok with that?

**Comment by dlford** [comment]
Appreciate the feedback folks, I'll re-visit this at a later date.

**Pull request closed without merging**